apps/frontend/src/app/5_pages/MarketMakingPage/components/AdjustAndDepositModal/hooks/useGetMinimumReturn.ts

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react';
+
+import { Decimal } from '@sovryn/utils';
+
+import { decimalic } from '../../../../../../utils/math';
+
+const SLIPPAGE_AMOUNT = 5;
```

Comment:
Where do you have this value from? Also, it should have a comment so we know how much is it (it could be 5%, 0.5%, ...).

I'd expect it to be named `DEFAULT_AMM_SLIPPAGE` and in some `constants` file.

Reply:
it's from D1